gradle.properties

```
@@ -1,8 +1,6 @@
-projectVersion=5.0.0-SNAPSHOT
+projectVersion=5.0.0-M5
```

```suggestion
projectVersion=5.0.0-SNAPSHOT
```

It is the release processes job to change this, no?